<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <column>
          <property name="text">
           <string>IP Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hostname</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Players</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Gamemode</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Language</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Password</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>OMP</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	
}</string>
         </property>
         <property name="text">
          <string>Open Multiplayer Server Browser</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelOnlinePlayers">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Online Players: &lt;span style=&quot; font-weight:600;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelOnlineServers">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Online Servers: &lt;span style=&quot; font-weight:600;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBoxSampServers">
         <property name="text">
          <string>sa-mp servers</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/imagesPrefix/icons/samp-icon.ico</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBoxOpenMpServers">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>open.mp servers</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/imagesPrefix/icons/open-mp-icon.ico</normalon>
          </iconset>
         </property>
         <property name="shortcut">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonRefresh">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/imagesPrefix/images/icon-refresh-24px.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>64</number>
         </property>
         <property name="placeholderText">
          <string>Search by Hostname, Gamemode, Language, Version</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAppearance">
    <property name="title">
     <string>Appearance</string>
    </property>
    <addaction name="actionTheme"/>
   </widget>
   <addaction name="menuAppearance"/>
  </widget>
  <action name="actionTheme">
   <property name="icon">
    <iconset>
     <normalon>:/imagesPrefix/images/icon-dark-64px.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources_rc.qrc"/>
 </resources>
 <connections/>
</ui>
